sdk/appconfiguration/app-configuration/test/README.md

```diff
@@ -0,0 +1,19 @@
+# Testing
+
+To test this project, make sure to first build it properly by following our [building instructions](https://github.com/Azure/azure-sdk-for-js/blob/master/CONTRIBUTING.md#building). Once the project is correctly built, you will be able to run the tests by following the [testing instructions](https://github.com/Azure/azure-sdk-for-js/blob/master/CONTRIBUTING.md#testing).
+
+To run the live tests, follow the [Integration Testing with live services](https://github.com/Azure/azure-sdk-for-js/blob/master/CONTRIBUTING.md#integration-testing-with-live-services) instructions to automatically set up the required resources with the proper configuration.
+
+The live tests in this project will use the resources created from the [ARM template](https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/appconfiguration/test-resources.json), which defines:
```

Comment:
> The live tests in this project will use the resources created from the [ARM template]

The "will use" here tells the reader that there is no other way to run the live tests and that they **have** to use the ARM template. The reader should know that they can chose to manually create the resource or use an already created one.

Also, we have already linked to the testing instructions, so we don't need to again link to the `Integration Testing with live services`. The above 2 paragraphs can be replaced with something like the below:


> You can bring your own Azure resources for the live tests to use or you can use our [script that creates the Azure resources](link to the New-TestResources.md file) using an ARM template. The below Azure resources are used by the tests in this project.

Reply:
> The "will use" here tells the reader that there is no other way to run the live tests and that they have to use the ARM template. The reader should know that they can chose to manually create the resource or use an already created one.

This is way harder than it seems because some of the tests require non trivial resources to make. The alternative to the ARM template is to go one by one documenting how to create all the resources needed to run live tests.

Reply:
> Also, we have already linked to the testing instructions, so we don't need to again link to the Integration Testing with live services. The above 2 paragraphs can be replaced with something like the below:

Sounds good!

Reply:
I have updated these files with all of your current feedback considered. Let me know how it looks! Also let me know if I've missed anything!